test/pylib/scylla_cluster.py

```diff
@@ -519,7 +533,7 @@ def __str__(self):
         host_id = getattr(self, 'host_id', 'undefined id')
         return f"ScyllaServer({self.server_id}, {self.ip_addr}, {host_id})"
 
-    def _write_config_file(self) -> None:
+    def write_config_file(self) -> None:
```

Comment:
I recall asking you why did you remove the `_` prefix for a private method on the other PR.
You said that it was by mistake and reverted the change.
Now you bring it back. Why?

Reply:
Because I lost my edit during a godzillion of rebases!-)